README.md

```diff
@@ -12,34 +12,19 @@ MIT Licensed.
 Usage
 -----
 
-Available flags for `unifi_exporter` include:
-
 ```
 $ ./unifi_exporter -h
 Usage of ./unifi_exporter:
-  -telemetry.addr string
-        host:port for UniFi exporter (default ":9130")
-  -telemetry.path string
-        URL path for surfacing collected metrics (default "/metrics")
-  -unifi.addr string
-        address of UniFi Controller API
-  -unifi.insecure
-        [optional] do not verify TLS certificate for UniFi Controller API (warning: please use carefully)
-  -unifi.password string
-        password for authentication against UniFi Controller API
-  -unifi.site string
-        [optional] description of site to collect metrics for using UniFi Controller API; if none specified, all sites will be scraped
-  -unifi.timeout duration
-        [optional] timeout for UniFi Controller API requests (default 5s)
-  -unifi.username string
-        username for authentication against UniFi Controller API
```

Comment:
I don't like removing options completely without deprecating them for one major version at least. As this is a `0.x` version, it should be fine though. However, I think it'd be even better to have the command line options override the options in the config file.

Reply:
I do agree that deprecating options in one fell swoop isn't usually the done way, however I'm personally really not a fan of passing authentication information in the command line and having them available for all to see in a ps output. Granted, nobody should be using an administrator account for read-only API calls, but it's still really bad practice. I'll think about it tomorrow and maybe surround the whole block in an if statement of some kind.

Reply:
I totally second removing the password. Having the rest of the options available would not hurt imho, but I came to the conclusion that you'd need a config file for the password in any case, so I'm fine with removing them.

Reply:
Having thought about it some more, I'd have to agree. If you have a config file to have the password, you might as well just have _only_ a config file as we can't perpetuate such glaring security risks.